src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.Common/Models/LayoutSets.cs

```diff
@@ -0,0 +1,15 @@
+using System.Collections.Generic;
+
+namespace Altinn.App.Common.Models
+{
+    /// <summary>
+    /// Layoutset
```

Comment:
Very minimalistic descriptions of class and property

Reply:
fixed